crowdstrike_rules/crowdstrike_detection_passthrough.yml

```diff
@@ -0,0 +1,65 @@
+AnalysisType: rule
+Filename: crowdstrike_detection_passthrough.py
+RuleID: Crowdstrike.Detection.passthrough
+DisplayName: Crowdstrike Detection Passthrough
+Enabled: true
+Severity: Medium
+LogTypes:
+  - Crowdstrike.DetectionSummary
+Tags:
+  - Crowdstrike
+Description: Crowdstrike Falcon has detected malicious activity on a host.
+Runbook: Follow the Falcon console link and follow the IR process as needed.
+DedupPeriodMinutes: 0
+SummaryAttributes:
+  - p_any_ip_addresses
+Tests:
+  -
+    Name: Low Severity Finding
+    ExpectedResult: true
+    Log:
+        {
+          "cid": "11111111111111111111111111111111",
+          "Technique": "PUP",
+          "ProcessId": 377077835340488700,
+          "AgentIdString": "00000000000000000000000000000000",
+          "DetectName": "NGAV",
+          "ComputerName": "macbook",
+          "ProcessStartTime": "2021-09-18 20:38:51Z",
+          "GrandparentCommandLine": "/sbin/launchd",
+          "MACAddress": "aa-00-00-00-00-00",
+          "CommandLine": "/Applications/app.app/Contents/MacOS/pup app",
+          "Objective": "Falcon Detection Method",
+          "Nonce": 1,
+          "SHA256String": "3333333333333333333333333333333333333333333333333333333333333333",
+          "ExternalApiType": "Event_DetectionSummaryEvent",
+          "PatternDispositionValue": 2176,
+          "DetectId": "ldt:00000000000000000000000000000000:222222222222222222",
+          "Severity": 2,
+          "PatternDispositionDescription": "Prevention/Quarantine, process was blocked from execution and quarantine was attempted.",
+          "SeverityName": "Low",
+          "MD5String": "33333333333333333333333333333333",
+          "EventUUID": "33333333333333333333333333333333",
+          "UserName": "bobert",
+          "FilePath": "/Applications/app.app/Contents/MacOS/",
+          "timestamp": "2021-09-18 20:38:52Z",
+          "ParentCommandLine": "/usr/libexec/runningboardd",
+          "DetectDescription": "This file is classified as Adware/PUP based on its SHA256 hash.",
+          "LocalIP": "192.168.1.1",
+          "ProcessEndTime": "1970-01-01 00:00:00Z",
+          "SHA1String": "0000000000000000000000000000000000000000",
+          "OriginSourceIpAddress": "",
+          "GrandparentImageFileName": "/sbin/launchd",
+          "MachineDomain": "",
+          "ParentImageFileName": "/usr/libexec/runningboardd",
+          "FalconHostLink": "https://falcon.us-2.crowdstrike.com/activity/detections/detail/00000000000000000000000000000000/222222222222222222?_cid=g04000bt4uaewmefdthny6fruqrhe2my",
```

Comment:
Just noticing CID at the end of this URL, not sure what it is but we might as well redact it

Reply:
Removed